api/src/main/resources/liquibase.xml

```diff
@@ -110,6 +110,32 @@
                                  referencedTableName="users" referencedColumnNames="user_id" />
     </changeSet>
 
+	<changeSet id="add_loinc_openmrs_concept_source_20200220" author="jecihjoy" dbms="mysql">
```

Comment:
Rename to changeSet id to **`add_loinc_openmrs_concept_reference_source_20200220`**